### DISCUSSION TITLE
Linkerd check failing for ExternalName

### DISCUSSION THREAD

**Discussion by mhonorio** [linkerd/linkerd2#9814] [General] [answered]
I recently upgraded Linkerd to 2.12.2 and I'm facing an issue with ExternalName service.

```
× cluster networks contains all services
    the Linkerd clusterNetworks ["10.0.0.0/8,100.64.0.0/10,172.16.0.0/12,192.168.0.0/16"] do not include svc infra/mongodb-cluster-externalname ()
    see https://linkerd.io/2.12/checks/#l5d-cluster-networks-pods for hints
```

As you may know, ExternalName in Kubernetes doesn't have any IP, so I cannot allow it in the clusterNetworks CIDR.

This blocks me to install Linkerd Viz.

Anyone else facing this issue?

**Comment by mateiidavid** [comment, marked as answer]
@mhonorio thanks for raising this. We've fixed this, you can check the details in the original issue, https://github.com/linkerd/linkerd2/issues/9661

The fix is present in the most recent edge at the time of writing (`edge-22.10.3`), and will likely be shipped in the next `stable-2.12.3` release.

**Reply by mhonorio** [reply]
Thanks @mateiidavid I'll upgrade to 2.12.3 as soon as it gets released

**Reply by seagullua** [reply]
It would be great to finally get the update. We can't use linkerd viz dashboard as it does not want to open the dashboard as a result of this error
